openml/datasets/data_feature.py

```diff
@@ -34,6 +36,7 @@ def __init__(  # noqa: PLR0913
         data_type: str,
         nominal_values: list[str],
         number_missing_values: int,
+        ontologies: list[str],
```

Comment:
I think this needs to have a default value as it is otherwise a breaking change?

Reply:
while technically it would make break code, I find it unlikely that people use this constructor outside our library. In the library, I think I have updated all constructors